File: 백준/Bronze/1110. 더하기 사이클/더하기 사이클.py
n = input()

if len(n) < 2:
    n = '0' + n

main = n

rst = 0

while 1:

    a = str(int(n[0]) + int(n[1]))
    n = n[1] + a[-1]  
    rst += 1

    if n == main:
        break

print(rst)

File: 백준/Bronze/1110. 더하기 사이클/더하기 사이클.cc
#include<bits/stdc++.h>
using namespace std;

int rst, n, origin;

int main(){

	cin >> n;
	
	origin = n;

	
	while(1){
		
		int a = (n / 10) + (n % 10);
		n = (n % 10) * 10 + (a % 10);
		
		rst++;
		
		if(origin==n) break;
	}
	
	cout << rst;
	
	return 0;
}